src/message.ts

```diff
@@ -59,7 +57,7 @@ interface DistributedMessageProperties {
    * since these are encoded as message headers; the supplied value
    * won't be mutated
    */
-  headers: MessageHeaders;
+  headers?: MessageHeaders;
```

Comment:
Why the Message interface changed? If it is necessary for new library the commit message should contain detailed information about this.

Reply:
I've brought library's old interface.